poa-odi-verification/rfc01_poa-odi-verification_embedded-odi-vc.md

```diff
@@ -39,148 +39,180 @@ Issued by a Company (EBSI TI) to a natural person. The subject contains natural
+The wallet needs to hold a valid ODI and POA verifiable credentials.
 
-## Embedding ODI as link
+## Include ODI when presenting POA
 
-PoA schema extends the base [EBSI Attestation schema](https://api-pilot.ebsi.eu/trusted-schemas-registry/v2/schemas/0xeb6d8131264327f3cbc5ddba9c69cb9afd34732b3b787e4b3e3507a25d3079e9)
-which defines the `evidence` property (see the [VC DM specification](https://www.w3.org/TR/vc-data-model-2.0/#evidence) for more details). The evidence field will be used
+The Verifiable Presenation will contain both the ODI and POA, the wallet should include the ODI if present when returning a POX.
 to add a link to ODI VP that is published online. The evidence field should also add digest property so that the verifier may check if the linked document is the same.
 
 ### Prerequisites
 
 * ODI VP published via publicly available URL
 
-### PoA Issuance
+### PoA Presentation
 
-Before the issuance process the issuer needs to know the ODI VP public URL. Ideally, the issuer should check if the ODI subject points to the issuer wallet DID to make sure
-the PoA credential is issued with correct ODI VP. The issuer constructs the `evidence` field containing the link to the ODI VP and digest property
-containing a hash of the ODI VP.
+When requesting the POA the requester should also optionally request the ODI. Using a presentation definition similar to below: 
+```json
+{
+  "id": "request-embedded-poa",
+  "input_descriptors": [
+    {
+      "id": "poa",
+      "name": "A POA",
+      "purpose": "Check your Power of Attorney (POA)",
+      "constraints": {
+        "fields": [
+          {
+            "path": [
+              "$.type"
+            ],
+            "filter": {
+              "type": "array",
+              "contains": {
+                "type": "string",
+                "pattern": "PowerOfAttorney"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "id": "oid",
+      "name": "Organizational Digital Identifier (ODI)",
+      "purpose": " ensure the PoA holder may act on behalf of a company",
+      "constraints": {
+        "fields": [
+          {
+            "path": [
+              "$.type"
+            ],
+            "optional": true,
+            "filter": {
+              "type": "array",
+              "contains": {
+                "type": "string",
+                "pattern": "OrganizationalDigitalIdentifier"
+              }
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+### 
 
-#### Example of PoA VC with ODI VP linked:
+#### Example of PoA VC with an included ODI as JWT:
 
 ```json
 {
   "@context": [
-    "https://www.w3.org/2018/credentials/v1"
+      "https://www.w3.org/2018/credentials/v1"
   ],
-  "credentialSchema": {
-    "id": "https://api-pilot.ebsi.eu/trusted-schemas-registry/v3/schemas/z2sbTT23X2zfsdMCPFMC7GiUwyHE91Lg9BpeEA5uqtT9s",
-    "type": "FullJsonSchemaValidator2021"
+  "type": ["VerifiablePresentation"],
```

Comment:
Could include another type in this array `["VerifiablePresentation", "IncludedODI"]` or `["VerifiablePresentation", "LinkedODI"]` name is not good but indicates to the validator to look for the ODI in the VP or for a linked ODI